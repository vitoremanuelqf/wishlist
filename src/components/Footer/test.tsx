import { render, screen } from '@testing-library/react'

import { Footer } from '.'

describe('<Footer />', () => {
  it('should render the copyright message in the footer', () => {
    const { container } = render(<Footer />)

    expect(
      screen.getByText(/Wishlist â€¢ 2024 | Todos os direitos reservados/i),
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
