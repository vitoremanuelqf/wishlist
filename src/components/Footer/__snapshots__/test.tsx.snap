// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render the copyright message in the footer 1`] = `
<footer
  class="mt-auto flex h-auto w-full justify-center bg-[#5b2b84]"
>
  <div
    class="flex h-auto w-full max-w-7xl items-center justify-center gap-2 px-4 py-2 sm:px-8"
  >
    <p
      class="text-center text-xs text-white"
    >
      Wishlist • 2024 | Todos os direitos reservados
    </p>
  </div>
</footer>
`;

exports[`<Footer /> should render the heading 1`] = `
<footer
  class="mt-auto flex h-auto w-full justify-center bg-[#5b2b84]"
>
  <div
    class="flex h-auto w-full max-w-7xl items-center justify-center gap-2 px-4 py-2 sm:px-8"
  >
    <p
      class="text-center text-xs text-white"
    >
      Wishlist • 2024 | Todos os direitos reservados
    </p>
  </div>
</footer>
`;
